packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -0,0 +1,412 @@
+                                                        files: df.files,
+                                                        error_message: df.error_message,
+                                                    });
+                                                }}
+                                                getSocket={WS.getSocket}
+                                                files_description={<FilesDescription />}
+                                                examples={<CommonMistakeExamples />}
+                                            />
+                                        </FormBodySection>
+                                    </FormBody>
+                                </ThemedScrollbars>
+                                {is_qualified_for_cfd_modal ? (
+                                    <Modal.Footer has_separator>
+                                        <FormSubmitButton
+                                            is_disabled={
+                                                isSubmitting ||
+                                                !!hasFieldsErrors(errors, values) ||
+                                                (document_file.files && document_file.files.length < 1) ||
+                                                !!document_file.error_message
+                                            }
+                                            label={localize('Save and submit')}
+                                            is_absolute={isMobile()}
+                                            is_loading={isSubmitting}
+                                            form_error={status?.msg}
+                                            has_cancel
+                                            onCancel={onCancel}
+                                            cancel_label={localize('Back')}
+                                        />
+                                    </Modal.Footer>
+                                ) : (
+                                    <FormFooter className='account-form__footer-poa'>
+                                        {status?.msg && <FormSubmitErrorMessage message={status.msg} />}
+                                        <Button
+                                            className='account-form__footer-btn'
+                                            type='submit'
+                                            is_disabled={
+                                                isSubmitting ||
+                                                !!hasFieldsErrors(errors, values) ||
+                                                (document_file.files && document_file.files.length < 1) ||
+                                                !!document_file.error_message
+                                            }
+                                            has_effect
+                                            is_loading={form_state.is_btn_loading}
```

Comment:
Can't we use `isSubmitting` from formik to handle the loading state of button?

Reply:
Yes we should have, We need to eliminate the whole `form_state` state from the component

Reply:
should I refactor it in this PR? @shahzaib-deriv , @likhith-deriv